R/install_greta_deps.R

```diff
@@ -4,15 +4,14 @@
 #'   these are TF 2.15.0, TFP 0.23.0, and Python 3.10. These Python modules
 #'   will be installed into a conda environment named "greta-env-tf2".
 #'
-#'   To see installation notes or errors, there isn't an argument to specify,
-#'     instead you will need to specify and environment variable,
-#'     `GRETA_INSTALLATION_LOG`, with
+#'   You can specify an environment variable to write a logfile to a specific
+#'     location with `GRETA_INSTALLATION_LOG` using
 #'     `Sys.setenv('GRETA_INSTALLATION_LOG'='path/to/logfile.html')`. Or use
 #'     [greta_set_install_logfile()] to set the path, e.g.,
-#'     `greta_set_install_logfile('path/to/logfile.html')`. You can also skip
-#'     the restarting of R and use [write_greta_install_log()]. If you have
-#'     not specified the `GRETA_INSTALLATION_LOG` environmental variable, the
-#'     installation notes will indicate how to use [write_greta_install_log()].
+#'     `greta_set_install_logfile('path/to/logfile.html')`. By default it uses
+#'     `tools::R_user_dir("greta")`. To see installation notes or errors, after
```

Comment:
`tools::R_user_dir("greta")` is a folder so what's the file path?

Reply:
Good point!